### ISSUE TITLE
Add missing dependency to Ubuntu PPA package

### ISSUE THREAD

**Issue by mildmojo** [Ulauncher/Ulauncher#551]
On a fresh ulauncher install from the recommended PPA in Linux Mint 18.3 (based on Ubuntu xenial), the app fails to launch with this error:

```
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 27, in <module>
    from ulauncher.main import main
  File "/usr/lib/python3/dist-packages/ulauncher/main.py", line 22, in <module>
    from ulauncher.ui.windows.UlauncherWindow import UlauncherWindow
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/UlauncherWindow.py", line 40, in <module>
    from ulauncher.ui.windows.PreferencesUlauncherDialog import PreferencesUlauncherDialog
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 10, in <module>
    gi.require_version('WebKit2', '4.0')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 106, in require_version
    (namespace, version))
ValueError: Namespace WebKit2 not available for version 4.0
```

Searching for this kind of error, I found a Stack Overflow question that led me to installing `gir1.2-webkit2-4.0`, which fixed the issue:

```
sudo apt install gir1.2-webkit2-4.0
```

I suggest adding that package as a dependency in the Ubuntu package in your PPA.

**Comment by friday** [comment]
It's not missing, but rather specified wrong: https://github.com/Ulauncher/Ulauncher/blob/35effa5328ecf6575ecf660023c4af477a052a89/debian/control#L23

We should remove gir1.2-webkit2-3.0 from here.

**Comment by friday** [comment]
Should be fixed in f0905b9 and will be released in 5.14.0
